App.tsx

```diff
@@ -55,15 +56,41 @@ function ScreenNavigators() {
 
 const MyComponent = () => {
   const state = useSelector(state => state.login);
+  const loadingStatus = useRef('stoped');
   const {loading} = state;
+
+  const indicatorTop = useRef(new Animated.Value(0)).current;
+  useEffect(() => {
+    if (loading && loadingStatus.current === 'stoped') {
+      loadingStatus.current = 'started';
+      Animated.timing(indicatorTop, {
+        toValue: 50,
+        duration: 1000,
+      }).start();
+    } else {
+      loadingStatus.current = 'stoped';
+      indicatorTop.setValue(0);
+    }
+  }, [indicatorTop, loading]);
   return (
     <NavigationContainer>
       <Drawer.Navigator drawerPosition={'left'} drawerContent={SideMenu}>
         <Drawer.Screen name="ScreenNavigators" component={ScreenNavigators} />
       </Drawer.Navigator>
       {loading === true ? (
         <View style={styles.loadingWrapper}>
-          <ActivityIndicator size="large" color="#0000ff" animating={true} />
+          <Animated.View
+            style={[
+              styles.loadingAniWrapper,
+              {
+                top: indicatorTop.interpolate({
```

Comment:
Please alwayse try to animated non layout properties. ` transform: [{ translateY }]`